src/main/kotlin/it/pagopa/wallet/services/MigrationService.kt

```diff
@@ -186,6 +186,18 @@ class MigrationService(
                     status = WalletStatusDto.CREATED,
                     paymentMethodId = paymentMethodId,
                     applications = listOf(application),
+                    clients =
+                        Client.WellKnown.values().associateWith { clientId ->
```

Comment:
Cause this code is used by different services, we can push it into domain layer by refactoring into something like:
`Client.WellKnown.fromOnboardingChannel(onboardingChannel)` or `Client.fromOnboardingChannel(onboardingChannel)`.

What do you think?

Reply:
Not necessary anymore now that we enable wallets for all well-known clients :)